README.md

```diff
@@ -196,9 +196,8 @@ Format and push a STAC collection.json file to a GitHub repository. Used by the
 
 #### Example
 
-Environment variables:
-GIT_AUTHOR_NAME="Example User"
-GIT_AUTHOR_EMAIL="<example@example.com>"
+Environment variable:
+GITHUB_API_TOKEN="pat_token_string"
```

Comment:
The way GitHub renders Markdown this [ends up on a single line](https://github.com/linz/argo-tasks/blob/a7a1b3e6c654b29650f5eaae09e91062b78d3e46/README.md):

> ![image](https://github.com/linz/argo-tasks/assets/168301/3ae036fd-5e48-462e-91e4-34df75763ff0)

Is that intentional? Also, do you want to `export` the variable so it's available for sub-shells?

[Tokens can be created](https://github.com/settings/tokens) three links away from the home page, so it would also be useful to link directly to it.

Reply:
I'll split this readme change into a separate PR as there are some issues to address about what audience we should document this for.